demo/index.html

```diff
@@ -40,17 +40,12 @@
           Prism.highlightAll();
         });
     </script>
-    <script type="module" src="../index.js"></script>
+    <script type="module" src="../index.js" data-demo-script="true"></script>
     <script type="module">
-      import { registerComponent } from "../index.js"
+      import {registerComponent} from '../dist/auro-accordion__bundled.js';
```

Comment:
I don't think we should be using the bundle here. Doing so means you have to run a full build for every small UI code change and slows down development. This page only gets used by us during development. While we should test the bundles, let's not default to using this file.

Reply:
I made this change so that the demo page could work when deployed to surge.sh. Unfortunately at the moment the script we wrote to update dependencies for surge.sh only updates the `script src=""` tags and not these `type="module` tags.

Since this only impacts the custom registration tho, I could return this to use `index.js` and I could open an issue with `auro-library` to update these examples to pull from the bundle as well as the other scripts? How do we feel about that strategy moving forward?

Reply:
@jason-capsule42 New issue created [`auro-library#20`](https://github.com/AlaskaAirlines/auro-library/issues/20)